GUI.py

```diff
@@ -0,0 +1,641 @@
+import wx
+import wx.lib.agw.aui as aui
```

Comment:
Remove means - remove, not comment it =)
Your code is overloaded by comments. I think it should be removed from code. Keep only what you use. Otherwise just do another function or generalize current function.